### DISCUSSION TITLE
Usage of Max7219 to drive individual LED's

### DISCUSSION THREAD

**Discussion by cougar2106** [MobiFlight/MobiFlight-Connector#596] [Ideas]
It is already possible to use the 7-segment method of the MAX7219
![MAX7219-7Segment](https://user-images.githubusercontent.com/31365414/144041657-0182331c-3d02-475d-a7ec-721202aa2cfb.jpg)

As I did not find the option (yet) in MobiFliht, would it be possible to use a future version of MobiFlight with the MAX7219 (& LedControl Library?) to have the option to address individual LED's via the Row / Column registers of the MAX7219?
![MAX7219-Matrix](https://user-images.githubusercontent.com/31365414/144040899-ebaa6b3f-8893-4428-9ad5-b29b3dcc38b4.jpg)
That way we could significantly increase the number of LED per Arduino, as 8 MAX7219 can be cascaded to control **512** LED's.
Thanks for taking it into consideration.

**Comment by DocMoebiuz** [comment]
We just removed this from the library. It was using 64 byte of memory. Not sure if we really want to add this again. We have added the output shift register to drive LEDs.

**Comment by elral** [comment]
I would also stay with the output shift register, they can achieve the same but are easier to connect (no matrix required) or more flexible in regard of numbers used (1 / 2 / ... shift registers).
